src/decorations.ts

```diff
@@ -1,86 +1,41 @@
-import { window, OverviewRulerLane, DecorationRangeBehavior } from 'vscode';
+import {
+  window,
+  OverviewRulerLane,
+  DecorationRangeBehavior,
+  ExtensionContext,
+  TextEditorDecorationType,
+} from 'vscode';
 
-export function failingItName() {
+function createTestStateDecoration(
```

Comment:
good move to consolidate the functions. 👍

Reply:
thx